src/Catalyst.Core.Modules.Kvm/DeltaExecutor.cs

```diff
@@ -469,40 +406,31 @@ private bool ValidateDeltaGasLimit(PublicEntry entry, ExecutionEnvironment env,
             ITxTracer txTracer)
         {
             if (_logger.IsEnabled(LogEventLevel.Verbose))
-            {
                 _logger.Verbose("Intrinsic gas calculated for {entry}: {intrinsicGas}", entry, intrinsicGas);
-            }
 
-            if (entry.GasLimit >= intrinsicGas)
-            {
-                return true;
-            }
-            
-            TraceLogInvalidTx(entry, $"GAS_LIMIT_BELOW_INTRINSIC_GAS {entry.GasLimit.ToString()} < {intrinsicGas.ToString()}");
+            if (entry.GasLimit >= intrinsicGas) return true;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4562326)